synapse/handlers/register.py

```diff
@@ -666,7 +678,8 @@ async def register_device(
         is_guest: bool = False,
         is_appservice_ghost: bool = False,
         auth_provider_id: Optional[str] = None,
-    ) -> Tuple[str, str]:
+        should_issue_refresh_token: bool = False,
+    ) -> Tuple[str, str, Optional[int], Optional[str]]:
```

Comment:
Is it OK to return a bunch of things in a tuple here, or should I better return some struct/dict?

Reply:
an attr.s would probably be nicer; it's ok though.